Singularity/Singularity/Screen/ELogEventType.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Runtime.Serialization;
+
+namespace Singularity.Screen
+{
+    public enum ELogEventType
+    {
+        [EnumMember(Value = "Unit attacked")]
+        UnitAttacked,
+
+        [EnumMember(Value = "Platform attacked")]
+        PlatformAttacked,
+
+        [EnumMember(Value = "Platform built")]
+        PlatformBuilt,
+
+        [EnumMember(Value = "unit built")]
```

Comment:
for consistency id make make the words start with caps "Unit Built"

Reply:
or "Unit built"

Reply:
definitely - fixed